packages/amplify-category-storage/src/index.ts

```diff
@@ -41,8 +44,11 @@ export {
   s3RemoveStorageLambdaTrigger,
 } from './provider-utils/awscloudformation/service-walkthroughs/s3-resource-api';
 
+/**
```

Comment:
nit: prefer to disable this lint rule for the file rather than have empty comment blocks